fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -1067,6 +1063,7 @@ private FHIRPersistenceNotSupportedException buildNotSupportedException(String m
     }
 
     /**
+     * Read the current version of the resource from the database.
```

Comment:
This is an implementation of the public interface...I don't think we should override the javadoc.
If we want to add additional info to the javadoc which is specific to this implementation, we can use `@implNote`.

Reply:
I don't think `@throws FHIRPersistenceResourceDeletedException` overrides the interface's javadoc, but if does then we should probably move that to the javadoc on the interface level.

Reply:
agreed. Removed the comment block